pkg/controller/registry/reconciler/configmap.go

```diff
@@ -290,9 +290,9 @@ func (c *ConfigMapRegistryReconciler) EnsureRegistryServer(logger *logrus.Entry,
 
 	if source.Spec.SourceType == v1alpha1.SourceTypeConfigmap || source.Spec.SourceType == v1alpha1.SourceTypeInternal {
 		// fetch configmap first, exit early if we can't find it
-		configMap, err := c.Lister.CoreV1().ConfigMapLister().ConfigMaps(source.GetNamespace()).Get(source.Spec.ConfigMap)
+		configMap, err := c.OpClient.KubernetesInterface().CoreV1().ConfigMaps(source.GetNamespace()).Get(context.TODO(), source.Spec.ConfigMap, metav1.GetOptions{})
```

Comment:
> ConfigMaps provided for the internal source type are user-created and
won't have our labels, so we need to use a live client to fetch them.

I feel like this would be a useful comment to have inline in the code - WDYT?

Reply:
Makes sense, done